tests/test_connector.py

```diff
@@ -3299,6 +3301,71 @@ async def test_connect_reuseconn_tracing(
     await conn.close()
 
 
+@pytest.mark.parametrize(
+    "use_proxy,use_proxy_headers,use_auth,expect_proxy_auth_header",
+    [
+        (True, True, True, True),
+        (True, True, False, False),
```

Comment:
I wonder if it makes more sense to use a variable with 3 states...
Or atleast make this one False, to better match what the test actually does (i.e. proxy headers does nothing if use_auth is False):
```suggestion
        (True, False, False, False),
```